pages/linux/trash.md

```diff
@@ -1,32 +1,32 @@
 # trash
 
-> A CLI for managing your trashcan / recycling bin.
+> A CLI for managing the trashcan / recycling bin.
 > More information: <https://github.com/andreafrancia/trash-cli>.
 
-- Delete a file (send to trash):
+- Delete a file and send it to the trash:
 
 `trash {{path/to/file}}`
 
-- List files in trash:
+- List all files in the trash:
 
 `trash-list`
```

Comment:
Does it need the - or can they be separate.

Reply:
It can't be separated. All those are technically different commands.